packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesConfiguration.java

```diff
@@ -47,6 +49,10 @@ public Uri getUpdateUrl() {
     return mUpdateUrl;
   }
 
+  public Map<String, String> getRequestHeaders() {
+    return mRequestHeaders;
+  }
```

Comment:
Unless we need to distinguish between "headers haven't been determined yet" and "the headers were set but there were none", I'd just make this always be a non-null Map and that way all the code that consumes this doesn't need to check if the map is null (it just needs to handle empty maps).